README.md

```
@@ -97,13 +99,13 @@ Edit your `~/.cursor/mcp.json` file with the following contents:
 Start the Argo CD MCP server from the binary after running `task install`:
 
 ```
-argocd-mcp http --argocd-url=<url> --argocd-token=<token> --debug=<true|false> --port=<port>
+argocd-mcp http --argocd-url=<url> --argocd-token=<token> --debug=<true|false> --listen=<[host]:port>
 ```
 
 Or start the Argo CD MCP server as a container after running `task build-image`:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)